views/breach-detail.hbs

```diff
@@ -3,7 +3,7 @@
   <main id="breach-detail" class="breach-detail clear-header bg-white flx jst-cntr" data-page-label="Breach Detail">
     <div class="mw-700 detail-section flx-cntr flx flx-col">
       <div class="breach-detail-logo-wrapper">
-        <img class="breach-detail-logo breach-logo" alt="{{ breach.Name }} logo" src="/img/logos/{{ breach.LogoPath }}" />
+        <img class="breach-detail-logo breach-logo" alt="{{ breach.Name }} logo" src="{{ ../LOGOS_ORIGIN }}/img/logos/{{ breach.LogoPath }}" />
```

Comment:
Unrelated to life, but should this be `{{ breach.Name }}` or `{{ breach.Title }}`?